### DISCUSSION TITLE
Extension to show whole code on the right side

### DISCUSSION THREAD

**Discussion by weissenbacherpwc** [microsoft/vscode-discussions#889] [Extension Show and Tell]
Hi,

I have a new Laptop so I needed to install all extensions in VS Code again. However I can't remember how the extension was called where you see the whole code at the right side (a bit transparent) when scrolling? This made it easier to jump to specific parts of the code. Does anyone remember how the etension/setting was called?

**Comment by gjsjohnmurray** [comment]
Are you sure this wasn't just the built-in minimap feature of VS Code?

**Reply by weissenbacherpwc** [reply]
Thats it, thank you!

**Comment by nickdavila** [comment]
I forgot the name and this helped me, thanks for posting!
